autopilot/model-explainability/explaining_customer_churn_model.ipynb

```diff
@@ -90,7 +90,7 @@
    "metadata": {},
```

Comment:
<p>What's the benefit of it over the existing one?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1967/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
If you're referring to the job name `<` and `>` fail the regex validation